torch_xla/csrc/data_ops.h

```diff
@@ -49,6 +49,9 @@ xla::XlaOp BuildMaskedFillScalar(xla::XlaOp input, xla::XlaOp mask,
 std::vector<int64_t> BuildSqueezedDimensions(
```

Comment:
The above one should be a special case of the 2nd one, shall we just keep the second one?